module-1/lab-numpy/your-code/main.py

```diff
@@ -109,13 +179,16 @@
 print(d_max, d_min, d_mean)
 
 
+# In[22]:
+
 
 #15. Now we want to label the values in d. First create an empty array "f" with the same shape (i.e. 2x3x5) as d using `np.empty`.
 
 f = np.empty([2, 3, 5])
```

Comment:
An additional option to np.empty could be initialize the array as a zeros array and then populate the values as indicated in the exercise. 
f = np.zeros(shape=(2,3,5)). It's another way!!